### ISSUE TITLE
Mobile Validation missing validation option buttons

### ISSUE THREAD

**Issue by daotyl000** [ProjectSidewalk/SidewalkWebpage#1881]
The agree, disagree, unsure buttons are missing from my mobile validations. It reappeared after I Skipped to the next pano

![IMG_9695](https://user-images.githubusercontent.com/28814007/64050852-1b644100-cb2e-11e9-8a8e-38f10b03cbda.PNG)

**Comment by htadeusiak** [comment]
Was this on the initially loaded pano or later @daotyl000 ?

**Comment by daotyl000** [comment]
@htadeusiak  This occurred on the initially loaded pano.

**Comment by maubinle** [comment]
I also saw this, and I think it happens when you’re accessing mobile sidewalk “within” another application. E.g. if you’re on Facebook (or in this case, slack) and you click the link it will open a browser window “within” the parent application. (You can tell this by the “done” UI top and bottom.)

**Comment by jonfroehlich** [comment]
I have also experienced this (by visiting the Project Sidewalk link via our Twitter post) but interestingly, it's seemingly inconsistent. For example, here's the broken UX visited via Twitter:

![image](https://user-images.githubusercontent.com/1621749/64752646-2d9e9180-d4d5-11e9-8d25-6c5e3c637cd2.png)

And here it worked:

![image](https://user-images.githubusercontent.com/1621749/64752669-3c854400-d4d5-11e9-990e-6dec088940f7.png)

I don't think I did anything differently.

As I said in Slack, I think this is an important bug to solve as most of our mobile traffic will be coming from Facebook, Twitter, and other social media.

**Comment by misaugstad** [comment]
Looks like @ThatOneGoat ran into this issue the other day too (documented in #2092). They were using an iPhone 6. @ThatOneGoat were you accessing the site by clicking a link? Or typing the address directly into the address bar?

**Comment by ThatOneGoat** [comment]
When I was testing on my branch, it was just through connecting to my ipaddress and then getting redirected to the mobile landing page. When I tested the prod server, I got to the address via the link on google (just searched up "project sidewalk" and clicked the link to the page)

**Comment by jonfroehlich** [comment]
I still see this problem intermittently tbh.

**Comment by misaugstad** [comment]
Do you just refresh the page and it typically is fixed?

**Comment by ThatOneGoat** [comment]
Yeah. I also reopened safari (the browser I was using), but I don't think that really affected it.

**Comment by misaugstad** [comment]
The UI here has been redone, so I'm assuming that this has been fixed by now.
